tsconfig.test.json

```diff
@@ -5,7 +5,7 @@
     "moduleResolution": "node",
     "jsx": "react",
     "experimentalDecorators": true,
-    "allowSyntheticDefaultImports": false,
+    "allowSyntheticDefaultImports": true,
```

Comment:
I had to do this in the test one because even it was throwing a fit when trying to push. Not sure why it hates me